README.md

```diff
@@ -332,7 +332,7 @@ Then [save and exit the editor](https://vim.rtorr.com/). The updates will be app
 ## For Members and Maintainers
 
 ### How to add yourself as a reviewer/approver
-There is an OWNERS file present in each component folder [like this](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/spi/OWNERS), people mentioned in the file have the respective access to approve/review PR's.
+There is an OWNERS file present in each component folder [like this](https://github.com/redhat-appstudio/infra-deployments/blob/pre-kcp/components/spi/OWNERS), people mentioned in the file have the respective access to approve/review PR's.
```

Comment:
I don't think this needs to change, does it?  The OWNERS file in main is the source of truth.

Reply:
don't forget that the "future" `main` branch (currently `kcp`) won't contain SPI component for now. SPI hasn't been fully onboarded to kcp yet.
In addition, I don't want to make this `pre-kcp` branch dependent on the structure of the `main` one